Dockerfile

```diff
@@ -1,35 +1,19 @@
-FROM node:15 As development
+FROM node:15 As building
 LABEL org.opencontainers.image.source https://github.com/eddiehubcommunity/api
-
+ARG github_token  
 WORKDIR /usr/src/app
-
 COPY package*.json ./
-
-RUN npm install
-
+COPY .npmrc .npmrc
+RUN echo "//npm.pkg.github.com/:_authToken=${github_token}" > ~/.npmrc
+RUN npm install --ignore-scripts
+RUN rm -f .npmrc
 COPY . .
-
 RUN npm run build
 
-FROM node:15 as production
-
-ARG NODE_ENV=production
-ENV NODE_ENV=${NODE_ENV}
-ENV VERSION="v0.0.0"
 
-RUN --mount=type=secret,id=github_token \
-  cat /run/secrets/github_token
-RUN echo "//npm.pkg.github.com/:_authToken=$github_token" > ~/.npmrc
+FROM node:15 
 
 WORKDIR /usr/src/app
-
-COPY package*.json ./
-
-
-RUN npm install --production --ignore-scripts
-
-COPY . .
-
-COPY --from=development /usr/src/app/dist ./dist
-
+COPY --from=building /usr/src/app ./
```

Comment:
I think we dont need the whole folder, just the `node_modules` and the package.json

Reply:
I think only the `dist` folder is needed https://github.com/EddieHubCommunity/api/pull/116/files/4bcdb9beeeee18eb50b42febc521384dde68f81a#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L33

Reply:
As far as I know the node_modules arent bundled in the `dist`, but let me have a check

Reply:
But you are right we definitely need `dist` I forgot about that folder

Reply:
![image](https://user-images.githubusercontent.com/43843195/120077162-526fd980-c0a9-11eb-964e-2e7214af270d.png)
Just deleted `node_modules` and run the `start:prod` script.

Reply:
For production build, I think everything we need should be in `dist`.

Reply:
Let me try too

Reply:
Oh you are right, it is needed 👍 (I was thinking of Angular 🤦‍♂️ )